### ISSUE TITLE
[FEATURE] Add generic aria-description to end-user-generated media.

### ISSUE THREAD

**Issue by Atticus29** [Symbiota/Symbiota#3375]
# Notes

The aria-description should be composed from the caption, notes, whether the images shows an organism, whether the images shows label data, whether the images shows an annotation/identification label, whether the images shows typed or printed text, whether the images shows handwritten text, whether the images shows habitat, whether the images has a problem, whether the images shows a diagnostic character, whether the images shows an adult organism, whether the images shows an immature organism.

<img width="1582" height="944" alt="Image" src="https://github.com/user-attachments/assets/e8726462-654c-4d3d-94e5-dff28e823e01" />

# Required

## Summary

See https://www.w3.org/WAI/WCAG22/quickref/?versions=2.1#audio-description-or-media-alternative-prerecorded.

## Github Issue/Projects Fields

- [x] Populate label (e.g., enhancement, etc.)
- [x] Populate the github projects fields (e.g., status, feature description, priority, affected branch)
- [ ] For any new pages that will be generated by this feature, add those pages to the [Symbiota-Laravel github issues](https://github.com/Symbiota/Symbiota-Laravel/issues) with a "Page porting" tag. Link it to this issue.
- [ ] If relevant, add reference to original osTicket:

# Ideal

## User Stories/Detailed Outline of All of the Specs

Avoid specific implementation details here. This section is for what needs to be done, not how it should be done.

- As a super admin, I want to be able to ...
- As a collection admin, I want to be able to ...
- As a signed in user with no admin authorizations, I want to be able to ...
- As an anonymous user, I want to be able to ...
- Etc.

## Implementation Details

If any specifics have been ironed out during the feature planning, add them here.

## Acceptance Criteria

- [ ] Criterion 1
- [ ] Criterion 2
- [ ] ...

## Dependencies

- [ ] List any dependencies, blockers, or prerequisites
- [ ] Related issues or PRs that need to be completed first

## Documentation

- [ ] API documentation updates needed

## Additional Context

Add any other context, screenshots, mockups, or examples that would be helpful.
